packages/bpk-tokens/src/base/typography.json

```diff
@@ -262,6 +262,126 @@
       "value": "{!WHITE}",
       "type": "color",
       "category": "text-colors"
+    },
+    "TEXT_XS_FONT_SIZE": {
+      "value": "{!FONT_SIZE_SM}",
+      "type": "size",
+      "category": "font-sizes"
+    },
+    "TEXT_XS_LINE_HEIGHT": {
+      "value": "{!SPACING_MD}",
+      "type": "number",
+      "category": "line-heights"
+    },
+    "TEXT_XS_FONT_WEIGHT": {
+      "value": "400",
+      "type": "number",
+      "category": "font-weights"
+    },
+    "TEXT_XS_LETTER_SPACING": {
+      "value": "normal",
+      "type": "size",
+      "category": "letter-spacings"
+    },
+    "TEXT_SM_FONT_SIZE": {
+      "value": "{!FONT_SIZE_SM}",
+      "type": "size",
+      "category": "font-sizes"
+    },
+    "TEXT_SM_LINE_HEIGHT": {
+      "value": "{!SPACING_MD}",
+      "type": "number",
+      "category": "line-heights"
+    },
+    "TEXT_SM_FONT_WEIGHT": {
+      "value": "400",
+      "type": "number",
+      "category": "font-weights"
+    },
+    "TEXT_SM_LETTER_SPACING": {
+      "value": "normal",
```

Comment:
[iOS](https://developer.apple.com/documentation/uikit/nskernattributename?language=objc) for reference